### PR TITLE
Remove experimental note of tiflash disaggregated arch

### PR DESCRIPTION
**Pull request by JaySon-Huang**
### First-time contributors' checklist 

- [x] I've signed [**Contributor License Agreement**](https://cla-assistant.io/pingcap/docs-cn) that's required for repo owners to accept my contribution.

### What is changed, added or deleted? (Required)

Remove experimental note of tiflash disaggregated arch

### Which TiDB version(s) do your changes apply to? (Required)

**Tips for choosing the affected version(s):**

By default, **CHOOSE MASTER ONLY** so your changes will be applied to the next TiDB major or minor releases. If your PR involves a product feature behavior change or a compatibility change, **CHOOSE THE AFFECTED RELEASE BRANCH(ES) AND MASTER**.

For details, see [tips for choosing the affected versions (in Chinese)](https://github.com/pingcap/docs-cn/blob/master/CONTRIBUTING.md#版本选择指南).

- [x] master (the latest development version)
- [x] v7.4 (TiDB 7.4 versions)
- [ ] v7.3 (TiDB 7.3 versions)
- [ ] v7.2 (TiDB 7.2 versions)
- [ ] v7.1 (TiDB 7.1 versions)
- [ ] v7.0 (TiDB 7.0 versions)
- [ ] v6.5 (TiDB 6.5 versions)
- [ ] v6.1 (TiDB 6.1 versions)
- [ ] v5.4 (TiDB 5.4 versions)
- [ ] v5.3 (TiDB 5.3 versions)
- [ ] v5.2 (TiDB 5.2 versions)
- [ ] v5.1 (TiDB 5.1 versions)
- [ ] v5.0 (TiDB 5.0 versions)

### What is the related PR or file link(s)?

- This PR is translated from: https://github.com/pingcap/docs/pull/14859
- Other reference link(s):

### Do your changes match any of the following descriptions?

- [ ] Delete files
- [ ] Change aliases
- [ ] Need modification after applied to another branch 
- [ ] Might cause conflicts after applied to another branch

### CODE REVIEW

File: tiflash/tiflash-disaggregated-and-s3.md [hunk 1]
```diff
@@ -60,26 +56,7 @@ TiFlash 存算分离架构适用于高性价比的数据分析服务的场景。
...
-        "And": {
-            "Tags": [
-                {
-                    "Value": "tiflash_deleted", 
-                    "Key": "true"
-                }
-            ]
-        }
-    }
-    ```
```
Comment (JaySon-Huang) [inline tiflash/tiflash-disaggregated-and-s3.md:80]: Remove because tiflash can automatically set it

**Review by flowbehappy — APPROVED** [review]
lgtm

File: tiflash/tiflash-disaggregated-and-s3.md [hunk 2]
```diff
@@ -60,26 +56,7 @@ TiFlash 存算分离架构适用于高性价比的数据分析服务的场景。
...
-
-3. 确保 TiDB 集群中没有任何 TiFlash 节点。如果有，则需要将所有表的 TiFlash 副本数设置为 0，然后缩容掉所有 TiFlash 节点。比如：
+2. 确保 TiDB 集群中没有任何存算一体的 TiFlash 节点。如果有，则需要将所有表的 TiFlash 副本数设置为 0，然后缩容掉所有 TiFlash 节点。比如：
```
Comment (qiancai) [inline tiflash/tiflash-disaggregated-and-s3.md:59]:
```suggestion
2. 确保 TiDB 集群中没有任何存算一体架构的 TiFlash 节点。如果有，则需要将所有表的 TiFlash 副本数设置为 0，然后缩容掉所有 TiFlash 节点。比如：
```

**Review by hfxsd — APPROVED** [review]

File: tiflash/tiflash-disaggregated-and-s3.md [hunk 3]
```diff
@@ -50,7 +46,7 @@ TiFlash 存算分离架构适用于高性价比的数据分析服务的场景。
 
     也可以使用兼容 S3 的其他对象存储，比如 [MinIO](https://min.io/)。
 
-    TiFlash 使用的 S3 API 接口列表包括：
+    需要确保 TiFlash 部署的节点有以下 S3 API 的权限，TiFlash 使用的 S3 API 接口列表包括：
```
Comment (qiancai) [inline tiflash/tiflash-disaggregated-and-s3.md:49]:
```suggestion
    需要确保 TiFlash 部署的节点有以下 S3 API 的权限：
```

Comment (JaySon-Huang) [inline tiflash/tiflash-disaggregated-and-s3.md:49]:
```suggestion
    TiFlash 将使用以下 S3 API 接口进行数据读写，需要确保部署 TiFlash 的节点有这些接口的权限：
```

**Review by qiancai — APPROVED** [review]

File: tiflash/tiflash-disaggregated-and-s3.md [hunk 4]
```diff
@@ -194,7 +171,7 @@ TiFlash 存算分离架构适用于高性价比的数据分析服务的场景。
 
 ## 使用限制
 
-- TiFlash 不支持在存算一体架构和存算分离架构之间原地切换。在切换架构前，需要将原有 TiFlash 节点全部删除。
+- TiFlash 不支持在存算一体架构和存算分离架构之间原地切换。在切换架构前，需要将原有存算一体的 TiFlash 节点全部删除。
```
Comment (qiancai) [inline tiflash/tiflash-disaggregated-and-s3.md:174]:
```suggestion
- TiFlash 不支持在存算一体架构和存算分离架构之间原地切换。在切换架构前，需要将原有架构的 TiFlash 节点全部删除。
```

Comment (qiancai) [inline tiflash/tiflash-disaggregated-and-s3.md:174]:
```suggestion
- TiFlash 不支持在存算一体架构和存算分离架构之间原地切换。在切换到存算分离架构前，需要将原有存算一体架构的 TiFlash 节点全部删除。
```

**Pull request merged**